addons/wiki/static/wikiPage.js

```diff
@@ -105,6 +105,12 @@ function ViewWidget(visible, version, viewText, rendered, contentURL, allowMathj
                             self.rendered(self.renderMarkdown(rawContent));
                             $markdownElement.css('display', 'inherit');
                         }
+                        if (window.contextVars.node.viewOnlyLink) {
+                            $($markdownElement).children('p').children('img').each(function(idx, elm){
+                                elm.setAttribute('src', elm.src + '&view_only=' + window.contextVars.node.viewOnlyLink);
```

Comment:
This doesn't seem quite right. This will add the `view_only` query to *every* img tag in the markdown content (not just OSF embeds).

Why not modify the the function used to build the MFR in `markdown.js`?